src/dat/gui/GUI.js

```diff
@@ -455,6 +456,8 @@ const GUI = function(pars) {
   }
 };
 
+GUI.CustomController = CustomController;
```

Comment:
Let's not assign this to the `GUI` object. Instead we can export `CustomController` from the module, so it can be used as:

```js
import { GUI, controllers } from 'dat.gui';

class FooController extends controllers.CustomController {}
```
See https://github.com/dataarts/dat.gui/blob/master/src/dat/index.js.

Reply:
Sorry I don't understand you. I can not include
`import { GUI, controllers } from 'dat.gui';`
 line into webgl_custom_controller.html file.

Reply:
If you aren't using ES modules, it would just be: `dat.controllers.CustomController` in this case.

Reply:
Thanks. Now I want to continue researching of your replies and want to rewrite my code

Reply:
I have removed GUI.CustomController. See [Remove GUI.CustomController](https://github.com/anhr/dat.gui/commit/9c86835a7d0920a9cd855c5ec34250905a274071).
I want to continue researching of your replies and want to rewrite my code